pkg/kubelet/kubelet.go

```diff
@@ -1743,6 +1746,9 @@ func (kl *Kubelet) syncTerminatingPod(ctx context.Context, pod *v1.Pod, podStatu
 	// manager or refresh the status of the cache, because a successful killPod will ensure we do
 	// not get invoked again
 	if runningPod != nil {
+		// Stop probing runtime pods immediately
+		kl.probeManager.RemovePod(pod)
```

Comment:
I also think this line may be useless, the reason is:
1. a runningPod which is synced from runtime could not exists in podmanager, it has no chance to run `syncPod`, so it should has no probes running
2. a runningPod terminating would not update status manager, so it would not report probe results

Reply:
I suspect both are of you a right. I removed this line from the current patch.